docs/changelog.rst

```diff
@@ -15,7 +15,13 @@ Here we list a changelog of pybv.
 Current
 =======
 
-- Nothing yet
+API
+~~~
+- :func:`pybv.write_brainvision` now accepts a list of units for the ``unit`` parameter, which will scale each channel separately, by `Adam Li`_ (`#66 <https://github.com/bids-standard/pybv/pull/66>`_)
```

Comment:
I think we can also add a new point regarding the fact that we also support non Volt channels now for the `data` parameter, and that these are "detected" through passing a list of str to `unit`, where the respective entries differ from nV, mV, µV, uV, or V.

Also another point about uV now being auto converted to µV.